nodewatcher/modules/frontend/ip_space/static/js/draw_cidrs.js

```diff
@@ -0,0 +1,147 @@
+window.DrawCidr = class DrawCidr{
```

Comment:
Please also make unified style here. Two spaces for indentation. See our other JavaScript code for example of code style. In short: we use `;` at the end f lines. We use `'` for strings, except if user-facing strings. We put spaces around `{`. We always go with a new block into new line.